app/src/main/java/org/thoughtcrime/securesms/home/HomeActivity.kt

```diff
@@ -203,7 +225,8 @@ class HomeActivity : PassphraseRequiredActionBarActivity(),
         }
 
         // Set up empty state view
-        binding.createNewPrivateChatButton.setOnClickListener { showNewConversation() }
+        binding.emptyStateContainer.setContent { EmptyView(textSecurePreferences.isNewAccount()) }
```

Comment:
yes, I'd love to.

Reply:
But yeah, then it'd have to get funneled by `LoadingActivity` each through `PickDisplayNameActivity` to `MessageNotificationActivity` and then to `HomeActivity`... so that's a bit of boilerplate.

Reply:
done